Lib9c/Model/Character/Player.cs

```diff
@@ -130,17 +130,17 @@ EquipmentItemSetEffectSheet equipmentItemSetEffectSheet
         }
 
         public Player(
-            int level, 
-            CharacterSheet characterSheet, 
-            CharacterLevelSheet characterLevelSheet, 
+            int level,
+            CharacterSheet characterSheet,
+            CharacterLevelSheet characterLevelSheet,
             EquipmentItemSetEffectSheet equipmentItemSetEffectSheet
         ) : base(
             null,
             characterSheet,
             GameConfig.DefaultAvatarCharacterId,
             level)
         {
-            Exp.Current = 0;
+            Exp.Current = characterLevelSheet[level].Exp;
```

Comment:
These changes will cause a state mismatch. However, the `Player(int level, ...)` constructor only inherits from the `EnemyPlayer(int level, ...)` constructor, and this `EnemyPlayer(int level, ...)` constructor is not used. That's why I suggest deleting both constructors.

Reply:
These constructors are for tests!